amf-api-contract/shared/src/main/scala/amf/apicontract/client/platform/model/domain/bindings/ibmmq/IBMMQMessageBinding.scala

```diff
@@ -0,0 +1,51 @@
+package amf.apicontract.client.platform.model.domain.bindings.ibmmq
+
+import amf.apicontract.client.platform.model.domain.bindings.{BindingVersion, MessageBinding}
```

Comment:
add all ibmmq things in the same commit as it is confusing to have the support for IBMMQ in two separate commits.

Reply:
done, reorganized commits